owasp-top10-2017-apps/a6/stegonography/app/index.js

```diff
@@ -48,7 +48,7 @@ MongoClient.connect(url, function(err, db) {
 MongoClient.connect(url, function(err, db) {
     if (err) throw err;
     var dbo = db.db("stego");
-    var myobj = { username: "admin", password: "admin" };
+    var myobj = { username: "admin", password: "adm9wE633cQxuzC6spY6SkJin" };
```

Comment:
Changing the `admin`'s password to a stronger one is a must, but it still is hardcoded in the application's source code. 🙃

What do you think of maybe using environment variables to handle these credentials? By doing this, you'll still have the same functionalities as before, but with the advantage of clearing the code of any sensitive information. 🐼